test/e2e/runner/load.go

```diff
@@ -40,31 +41,39 @@ func Load(ctx context.Context, testnet *e2e.Testnet) error {
 		}
 
 		for w := 0; w < testnet.LoadTxConnections; w++ {
-			go loadProcess(ctx, txCh, chSuccess, n)
+			go loadProcess(ctx, txCh, chSuccess, chFailed, n)
 		}
 	}
 
-	// Monitor successful transactions, and abort on stalls.
-	success := 0
+	// Monitor successful and failed transactions, and abort on stalls.
+	success, failed := 0, 0
 	timeout := initialTimeout
 	for {
+		rate := log.NewLazySprintf("%.1f", float64(success)/time.Since(started).Seconds())
```

Comment:
shouldn't the rate be the total `float64(success+failed)/time.Since(started).Seconds()` to give a better accurate result? I'm assuming here this is the `txs/secs` rate

Reply:
`success/started` is the real load rate of transactions that were actually delivered since the load started, while `(success+failed)/started` is a constant value equal to the parameter `load_tx_batch_size` (the number of expected txs/second).